### PR TITLE
Use mux-player-react as video player

### PR DESCRIPTION
**Pull request by davekiss**
- [ ] Test query param `currentTime`
- [ ] Implement onError handler
- [ ] Track playing status via handler
- [ ] Poster should not be required as it it derived automatically in `mux-player-react`
- [ ] Implement onLoaded handler
- [ ] smoke test on mobile

### CODE REVIEW

**Comment by philcluff** [comment]
Love this @davekiss - question: Could we A/B test plyr vs mux-player-react for a period and validate on Data?

**Comment by davekiss** [comment]
@philcluff I would imagine so. What measurements do you have in mind?

**Comment by philcluff** [comment]
I'd start by measuring the top line Overall Viewer Score in Mux Data, and work from there!

File: components/video-player.tsx [hunk 1]
```diff
@@ -39,164 +9,23 @@ type Props = {
...
-        }
-      `}
-      </style>
-    </>
+    <div className='video-container'>
+      <MuxPlayer
+        playbackId={playbackId}
+        poster={poster}
+        muted
+        controls
```
Comment (cjpillsbury) [inline components/video-player.tsx:24]: controls shouldn't be necessary

File: components/video-player.tsx [hunk 2]
```diff
@@ -39,164 +9,23 @@ type Props = {
...
+        playsInline
```
Comment (cjpillsbury) [inline components/video-player.tsx:25]: do you know if this is necessary?
> A Boolean attribute indicating that the video is to be played "inline", that is within the element's playback area. Note that the absence of this attribute does not imply that the video will always be played in fullscreen.
  - From https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video#attr-playsinline

File: pages/v/[id]/embed.tsx [hunk 3]
```diff
@@ -119,7 +118,7 @@ const PlaybackEmbedded: React.FC<Props> = ({ playbackId, poster }) => {
       {errorMessage && <h1 className="error-message">{errorMessage}</h1>}
       {showLoading && <FullpageLoader text="Loading player" />}
       <div className="wrapper">
-        <VideoPlayer ref={videoRef} playbackId={playbackId} poster={poster} currentTime={startTime} onLoaded={() => setIsLoaded(true)} onError={onError} />
+        <VideoPlayer playbackId={playbackId} poster={poster} currentTime={startTime} onLoaded={() => setIsLoaded(true)} onError={onError} />
```
Comment (cjpillsbury) [inline pages/v/[id]/embed.tsx:121]: Shouldn't need "`poster`"

**Pull request closed without merging**